spec/pluck_to_tstruct_spec.rb

```diff
@@ -74,7 +74,37 @@ class WizardWithWandT < T::Struct
 
     def ==(other)
       return false unless other.is_a?(self.class)
-      name == other.name && house == other.house
+      name == other.name && house == other.house && wand_wood_type == other.wand_wood_type
+    end
+
+    def eql?(other)
+      self == other
+    end
+  end
+
+  class WizardWithDefaultParentEmailT < T::Struct
+    const :name, String
+    const :house, String
+    const :parent_email, String, default: "hagrid@hogwarts.com"
+
+    def ==(other)
```

Comment:
Not a blocker, but could you please see if there's a way to extract this common logic into a module, to make these classes easier to write? Ideally we'd do:

```ruby
class WizardWithDefaultParentEmailT < T::Struct
  include ComparableWizardStruct
  
  const :name, String
  # ... other consts ...
end
```

Reply:
Good call [`c193528` (#386)](https://github.com/chanzuckerberg/sorbet-rails/pull/386/commits/c1935284d347e9e68efeebc8685ec89a46328518)